tests/storage/storage.go

```diff
@@ -267,13 +266,16 @@ var _ = SIGDescribe("Storage", func() {
 						}
 						nfsPod = storageframework.InitNFS(targetImage, nodeName)
 						pvName = createNFSPvAndPvc(family, nfsPod)
-						ignoreWarnings = true
 					} else {
 						pvName = tests.DiskAlpineHostPath
 					}
 					vmi = newVMI(pvName)
 
-					tests.RunVMIAndExpectLaunchWithIgnoreWarningArg(vmi, 180, ignoreWarnings)
+					if storageEngine == "nfs" {
+						tests.RunVMIAndExpectLaunchIgnoreWarnings(vmi, 180)
```

Comment:
@EdDev I don't understand you comment. Do you refer to this not updating `vmi`? Would you elaborate what needs to be fixed?

Reply:
This one was not updating the VMI even before, but others did. I will comment on one example of this.